data/rubygems.yml

```diff
@@ -10,8 +10,10 @@ activerecord: rails/rails
 activesupport: rails/rails
 capistrano: capistrano/capistrano
 coffee-script-source: jashkenas/coffeescript
+ffi: ffi/ffi
```

Comment:
No longer needed to fix by https://github.com/ffi/ffi/pull/703. This problem will be fixed in the next `ffi` release.